### PR TITLE
Literal imports and some style tweaks.

### PR DESCRIPTION
**Pull request by FrankD412**
Some suggested style improvements. I didn't want to commit directly to your branch since I didn't know what you had in mind. I've changed some things to literal imports, but I don't think I caught them all.

### CODE REVIEW

File: samples/parameterization/itertools_pgen.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+from maestrowf.datastructures.core import ParameterGenerator
```
Comment (jwhite242) [inline samples/parameterization/itertools_pgen.py]: Should we rename this one lulesh_itertools_pgen.py perhaps, or something similar that makes it more clear what it's for?

Comment (FrankD412) [reply samples/parameterization/itertools_pgen.py]: Yeah, that's a good idea. Stand by...

Comment (FrankD412) [reply samples/parameterization/itertools_pgen.py]: I also had the thought, should we comment these?

File: samples/parameterization/np_cheb_pgen_pargs.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    angles = np.linspace(np.pi, 0.0, num_pts)
+    xpts = r*np.cos(angles) + r
+    ypts = r*np.sin(angles)
+
+    return xpts, ypts
+
+
+def get_custom_generator(env, **kwargs):
+    p_gen = ParameterGenerator()
```
Comment (jwhite242) [inline samples/parameterization/np_cheb_pgen_pargs.py:16]: So, more of a general example discussion point: now that these are standalone files in the repo and not just examples in the docs, should we go back and add proper docstrings to these functions?  i know bloats the example docs size with non-critical info, but undocumented functions drive me nuts when i run into them..

File: samples/parameterization/itertools_pgen.py (hunk 1 above)
Comment (jwhite242) [reply samples/parameterization/itertools_pgen.py]: funny, just commented with the same thought on one of the other examples

File: samples/parameterization/np_cheb_pgen_pargs.py (hunk 2 above)
Comment (FrankD412) [reply samples/parameterization/np_cheb_pgen_pargs.py:16]: I was going to make the same suggestion haha

File: samples/parameterization/itertools_pgen.py (hunk 1 above)
Comment (FrankD412) [reply samples/parameterization/itertools_pgen.py]: Do you want to handle that once this PR gets merged?

Comment (jwhite242) [reply samples/parameterization/itertools_pgen.py]: Sure

**Pull request merged**